### PR TITLE
made a few adjustments to the servelet

### PR DESCRIPTION
**Pull request by sheneska**

### CODE REVIEW

**Review by MattIrv — APPROVED** [review]
Looks good! Glad to hear you are enjoying SPS 👍

**Review by MattIrv — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
-@WebServlet("/hello")
+@WebServlet("/myportfolio")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>I learnt Javascript, HTML and front end/back end web development. </h1>");
```
Comment (MattIrv) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: 🥳 🥳 🥳

**Review by MattIrv — APPROVED** [review]
Looks good to me!

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java (hunk 1 above)
Comment (sheneska) [reply portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: :)))))

**Review by MattIrv — APPROVED** [review]

**Pull request merged**